cmd/fossa/cmd/test/test.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 var Timeout = "timeout"
+var NoFail = "nofail"
```

Comment:
[nit] I think we can do better than `nofail`, I like to make flags explicitly related to what the user is trying to accomplish. Something like `suppress`, `suppress-issues`, or `suppress-failure` sounds a little more in line with what the purpose of the flag is.